README.md

```diff
@@ -1,2 +1,7 @@
-# ruby-homework
-A repository I will use to solve my Ruby homework
+# ruby-homework-modified
+A repository I will use to solve my Ruby homework from ruby course [https://github.com/monorkin/learn.rb](URL)
+
+##My favorite programming languages
```

Comment:
Also, I didn't notice that you are missing a space here. If you look at the preview, you'll see that this line is not shown as a header because of that.